source/tasks/scheduleTask.ts

```diff
@@ -10,16 +16,62 @@ export const generateTaskSchedulerForInstallation = (installationID: number) =>
    * @param time
    * @param data
    */
-  const taskScheduler = (taskName: string, time: string, data: any) => {
+  const taskScheduler = async (taskName: string, time: string, data: any) => {
     const config: DangerFileTaskConfig = {
       taskName,
       data,
       installationID,
     }
 
     const sanitizedTime = time.replace("in ", "")
-    agenda.schedule(sanitizedTime, runDangerfileTaskName, config)
+    // If you're running on your own server, you can just call agenda
+    // but if you're not then you're going to need to make an API call
+    if (agenda) {
+      agenda.schedule(sanitizedTime, runDangerfileTaskName, config)
+    } else {
+      const settings = sandboxSettings!
+
+      const input = {
+        jwt: settings.perilJWT,
+        task: runDangerfileTaskName,
+        time: sanitizedTime,
+        data,
+      }
+      const query = `mutation {
+        scheduleTask(
+          ${objectToGraphQLInput(input)}
```

Comment:
this is the only thing I'm not certain on, mainly how it handles a bunch of weird json